src/js/components/List/List.js

```diff
@@ -562,6 +643,24 @@ const List = React.forwardRef(
                   }
                 }
 
+                let displayPinned;
+                if (isPinned) {
+                  boxProps = {
+                    direction: 'row',
+                    align:
+                      (defaultItemProps && defaultItemProps.align) || 'center',
+                    gap: 'medium',
+                  };
+                  displayPinned = (
+                    <Box direction="row" align="center" justify="end">
+                      <Box pad="small">
+                        <Pin viewBox="0 0 28 28" />
+                      </Box>
```

Comment:
Is this trying to size the icon and then add pad? Seems like you'd judy want some pad on box at 655 and a `size` on the icon.

Reply:
657 is reducing the size of the pin icon since it was too large compared to the arrow icons.

Reply:
You can use pixel sizes for the icon size property

Reply:
Okay I'll give that a shot tonight. I can't remember, but I feel like I tried putting in a pixel size and it wasn't working, but I'll double check tonight. I think I just tried `size="18px"` for example, but is it supposed to an object?

Reply:
Just a string. Here's an example:
![image](https://user-images.githubusercontent.com/12054362/181069267-8fa8d075-278c-4ea3-9c92-4a5b42fb6c6d.png)

Reply:
I think you should also add the icon and the size of the icon to the theme. You can see an example in base.js for list for the up down icons on line 1030

Reply:
Thank you so much for all of your help. I updated the icon based on your feedback and used the base theme for that.